doc/src/user_guide.rst

```diff
@@ -39,6 +39,16 @@ or greater. The backend requires:
 1. `CUDA <https://developer.nvidia.com/cuda-downloads>`_ >= 4.2
 2. `pycuda <http://mathema.tician.de/software/pycuda/>`_ >= 2011.2
 
+MIC Backend
+^^^^^^^^^^^
+
+The MIC backend targets Intel Xeon Phi co-processors. The backend
+requires:
+
+1. ICC >= 16.0
```

Comment:
After reporting this issue, Michael fixed it and pyMIC can be compiled with MPSS 3.6.1. As pointed in pyMIC web page, I confirmed the following version of compiler and libraries (greater than Intel Composer XE 2013SP1 and MPSS 3.3) work well.
ICC >= 14.0
Intel MKL >= 11.1
Intel MPSS >= 3.3
pymic >= 0.7 (post commit 4d8a2da)